frontend/shared-components/external-link/external-link.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styles from './external-link.module.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons'
+
+interface ExternalLinkProps { linkPath: string, children: React.ReactNode }
+
+export default function ExternalLink ({ linkPath, children }: ExternalLinkProps) {
+  return (
+    <a 
+      className={styles.externalLink} 
+      target="_blank"
+      rel="noopener noreferrer"
+      href={linkPath}
+      >
+        {children}
```

Comment:
Is this intended to return children somewhere else not currently shown in this PR? I only see this wrapping a hardcoded string above here...

Reply:
You're totally right, a string property would be fine